### DISCUSSION TITLE
Decker @ GitHub

### DISCUSSION THREAD

**Discussion by monofon** [decker-edu/decker#2] [Announcements]
Ich habe das Decker-Repo mal testweise nach GitHub migriert. 

https://github.com/decker-edu/decker

Schaut mal, ob irgendwas fehlt. GitHub hat für alle Nutzer deren Email es kannte, gleich den jeweiligen GitHub-Account mit den Commits verknüpft. (@marcerich @korwisi, wenn ihr eure Emailadresse, mit der ihr bisher committed habt, zu eurem GitHub-Account hinzufügt, wird das automatisch nachgezogen.)

Bei der Gelegenheit habe ich alle Branches ausser `master` und `develop` zurückgelassen. Wenn Ihr noch irgendwas, das mit soll, auf einem lokalen Branch rumliegenhabt, dann mergt das nach `develop` und comittet.

Ich schlage vor, wir fangen das erst an zu benutzen, wenn alle ihr OK gegeben haben.

Danach können wir dann die CI wieder nach und nach in Betrieb nehmen (Hat die letzten Monate ja eh nicht mehr funktioniert).

**Comment by marcerich** [comment]
Hi Henrik,

ich arbeite noch auf dem branch "resource-pack-transition-wue“. 

Soll ich den jetzt auf den develop branch auf GitHub mergen?

Marc

**Reply by monofon** [reply]
Ja.

Kann ich aber auch erledigen.
